libs/template/presets/storefront/experience/pages/cart-page.ts

```diff
@@ -10,15 +10,39 @@ export const cartPage: StaticComponent = {
   options: {
     data: {
       rules: [
-        {
-          layout: 'split-main',
-          padding: '30px 0',
-        },
+        { layout: 'split-main', padding: '30px 0' },
         { query: { breakpoint: 'sm' }, gap: '0' },
       ],
     },
   },
   components: [
+    {
+      type: 'oryx-content-text',
+      content: {
+        data: {
+          text: `
+          <oryx-icon type="shopping_cart" style="--oryx-icon-size: 40px;"></oryx-icon>
+          <p>Your shopping cart is empty</p><oryx-button>
+          <a href="/search">Shop now</a></oryx-button>`,
+        },
+      },
+      options: {
+        data: {
+          visibility: { hideByRule: 'CART.!EMPTY' },
```

Comment:
Should then cart-entries be hidden when `CART.EMPTY` ?

Reply:
no, we have that logic inside. I'm thinking to keep it there, as we likely move into skeleton design for the cart page. We'd ideally do the skeleton with existing (empty) ui components, just iterating over an empty ("start with") list of entries.